llvm/lib/Target/NVPTX/NVPTXIntrinsics.td

```diff
@@ -782,6 +922,15 @@ foreach dim = 1...5 in {
     defm TMA_TENSOR_PF_ # suffix : TMA_TENSOR_PREFETCH_INTR<dim, mode>;
   }
 }
+foreach dim = 3...5 in {
+  foreach mode = ["im2col_w", "im2col_w_128"] in {
+    defvar suffix = !toupper(mode) # "_" # dim # D;
```

Comment:
Nit: here and in other places make it  `"D"`, so it's clear that it's a string literal, similar to `"_"`.